packages/manager/cypress/e2e/core/cloudpulse/dbaas-widgets-verification.spec.ts

```diff
@@ -295,7 +323,29 @@ describe('Integration Tests for DBaaS Dashboard ', () => {
     });
 
     // Wait for all metrics query requests to resolve.
-    cy.wait(['@getMetrics', '@getMetrics', '@getMetrics', '@getMetrics']);
+    cy.get('@getMetrics.all')
+      .should('have.length', 4)
+      .each((xhr: unknown) => {
+        const interception = xhr as Interception;
+        const { body: requestPayload } = interception.request;
+        const { filters } = requestPayload;
+
+        // Validate filter for "dimension_1"
+        const dimension1Filter = filters.filter(
```

Comment:
```suggestion
        const dimension1Filter = filters.find(
```
This will return only one dimensionFilter and you can validate below